engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -649,7 +649,7 @@ protected void serviceStart() throws Exception
       addSSLConfigResource(config);
       WebApp webApp = WebApps.$for("stram", StramAppContext.class, appContext, "ws").with(config).start(new StramWebApp(this.dnmgr));
       LOG.info("Started web service at port: " + webApp.port());
-      appMasterTrackingUrl = NetUtils.getConnectAddress(webApp.getListenerAddress()).getHostName() + ":" + webApp.port();
+      appMasterTrackingUrl = NetUtils.getConnectAddress(webApp.getListenerAddress()).getAddress().getCanonicalHostName() + ":" + webApp.port();
```

Comment:
@PramodSSImmaneni Whether it is a better thing to do depends on a configuration and an environment. I saw issues with virtual environments/sandboxes where FQDN was not resolvable, while "localhost" works. The misconfiguration may be on a host side and on the DNS side as well.

I don't mind changing from getHostName() to getCanonicalHostName(), but it will be good to understand why getHostName() does not return FQDN even though it is expected to do reverse name lookup as well.